app/static/css/main.css

```diff
@@ -160,6 +164,9 @@ textarea {
 		float: none;
 		position: static;
 	}
+	.mobile-only {
```

Comment:
imo this kind of blurs the line of styling and content, I'd prefer to name the class what it actually is and then handle the non-displaying in the CSS, but also I don't really care